src/content/docs/tutorial-service-level-mgmt/establish-input-sli.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Establish an input performance SLI"
+metaDescription: "New Relic observability maturity series: our service level management guide shows you how to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders."
+redirects:
+freshnessValidatedDate: never
+---
+
+import apmAddInputServiceLevel from 'images/apm_screenshot-crop_add-input-service-level.webp'
+
+import apmSyntheticsSetup from 'images/apm_screenshot-crop_synthetics-setup.webp'
+
+We use synthetic monitoring to simulate connections from all over the world. If there's an interruption between the source point and the output, the synthetic checks will fail, affect in your service level objective scores, and trigger very critical alerts. We refer to this as input performance.
```

Comment:
```suggestion
We use synthetic monitoring to simulate connections from all over the world. If there's an interruption between the source point and the output, the synthetic checks will fail, affect your service level objective scores, and trigger very critical alerts. We refer to this as input performance.
```

Reply:
This is a little confusing because synthetic checks were a prereq for #1 in the series. Should this actually be the first step? i'm just a littlee fuzzy about why you would do this second, or what outcome a user is meant to get. 

- Are synthetic checks basically the SAT to measure your service levels? Or are they the data you use to create a service level... or are they both?
- "source point" and "output" seems very nebulous and nonspecific. Can we put this in tangible terms somehow? 
- Once I set this up, what am I supposed to be waiting for? Is there something I should check back, or is it more of an investment thing -- you set it up and forget about it / wait for alerts?

I think clarifying these might help make this doc seem more robust and fit into the story you're building